tools/ci_connector_ops/ci_connector_ops/pipelines/actions/environments.py

```diff
@@ -671,6 +672,16 @@ async def with_airbyte_java_connector(context: ConnectorContext, connector_java_
     return await finalize_build(context, connector_container)
 
 
+async def get_cdk_version_from_python_connector(python_connector: Container) -> Optional[str]:
+
+    pip_freeze_stdout = await python_connector.with_entrypoint("pip").with_exec(["freeze"]).stdout()
+    pip_dependencies = [dep.split("==") for dep in pip_freeze_stdout.split("\n")]
```

Comment:
No I don't think so because `pip freeze` actually outputs the installed python packages, that should all have a version after `==`. And even if a silly version was shipped to pypi and pinned to the connector it should not break this flow, it'll just label the connector with a "silly" but true version.